code/unit_tests/origins_tests.dm

```diff
@@ -0,0 +1,33 @@
+/datum/unit_test/cultures
+	name = "Cultures Test - All cultures shall be filled out"
+
+/datum/unit_test/cultures/start_test()
+	var/list/decl/origin_item/culture/all_cultures = decls_repository.get_decls_of_subtype(/decl/origin_item/culture)
+	for(var/decl/origin_item/culture/OC in all_cultures)
```

Comment:
```suggestion
	for(var/decl/origin_item/culture/OC as anything in all_cultures)
```

We don't want the unit test to silently ignore it if we somehow put a non-/decl/origin_item/culture in all_cultures.